hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -464,6 +469,8 @@ public void createBucket(
         .setStorageType(storageType)
         .setSourceVolume(bucketArgs.getSourceVolume())
         .setSourceBucket(bucketArgs.getSourceBucket())
+        .setQuotaInBytes(quotaInBytes)
+        .setQuotaInCounts(quotaInCounts)
```

Comment:
Do you need to verify whether `quotaInBytes` and `quotaInCounts` are valid? e.g. >= 0?

Reply:
Had added a verify method In RpcClient.